compiler/definitions/ir/nodes/r_wrap.py

```diff
@@ -28,8 +28,10 @@ def wrap_node(node):
 
     ## TODO: All arguments must be options, otherwise there must be
     ##       special handling in the wrap node2ast code. 
-    old_options_strs = [str(opt) for i, opt in node.com_options]
-    wrapped_command = " ".join([str(node.com_name)] + old_options_strs)
+    # print("options: ",  node.com_options.replace("\\", ""))
+    
+    old_options_strs = [opt.raw_str() for i, opt in node.com_options]
+    wrapped_command = " ".join([node.com_name.raw_str()] + old_options_strs)
     wrapped_command_arg = [(1, Arg(string_to_argument(f"\'{wrapped_command}\'")))]
```

Comment:
I would prefer not going back to string to go back to argument. This introduces an unnecessary fail point.

Could we maybe just put all of those in `Q` straight from `Arg`? Argument has an `ast_list` field that contains characters of the form https://github.com/andromeda/pash/blob/main/compiler/parser/ast_atd.atd#L52. 

If this does not work, we need to _at least_ write some comments here about this raw_str and what it does.

Reply:
Using Q didn't work because I want the type of quotes to be single ones so that they don't interfere with double quotes that exist in the command itself. I get bash -c "sed"s/T..:..:..//"" instead of 'sed"s/T..:..:..//"'